plugins/de/opendiabetes/vault/plugin/interpreter/dateInterpreter/help.md

```diff
@@ -0,0 +1,21 @@
+# DateInterpreter
+Version: 1.0.0  
+Classification: Interpreter
+
+Overview
+-----
+The DateInterpreter is a plugin which interprets VaultEntry dates containing only the entries that are specified in the given import period.
```

Comment:
Klingt irgendwie komisch, würde _... which interprets VaultEntry data so that only entries that lie in the specified import period remain._ vorschlagen